buildenv/jenkins/README.md

```diff
@@ -274,438 +158,91 @@ Having a complicated regex in the pull request trigger is what allows us to laun
-        - Triggers
-            - `Build-JDK9-linux_x86-64_cmprssptrs`
-            - `Test-Sanity-JDK9-linux_x86-64_cmprssptrs`
-            - `Test-Extended-JDK9-linux_x86-64_cmprssptrs`
-    - Trigger:
-        - build periodically, @midnight
-
-- Pipeline-Build-Test-JDK10-linux_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Pipeline-Build-Test-JDK10-linux_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Pipeline-Build-Test-JDK10-linux_x86-64_cmprssptrs)
-    - Description:
-        - Compile and Test java10 Sanity & Extended
-        - Triggers
-            - `Build-JDK10-linux_x86-64_cmprssptrs`
-            - `Test-Sanity-JDK10-linux_x86-64_cmprssptrs`
-            - `Test-Extended-JDK10-linux_x86-64_cmprssptrs`
-    - Trigger:
-        - build periodically, @midnight
-
-- Pipeline-Build-Test-JDK8-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Pipeline-Build-Test-JDK8-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Pipeline-Build-Test-JDK8-win_x86-64_cmprssptrs)
-    - Description:
-        - Compile and Test java8 Sanity & Extended
-        - Triggers
-            - `Build-JDK8-win_x86-64_cmprssptrs`
-            - `Test-Sanity-JDK8-win_x86-64_cmprssptrs`
-            - `Test-Extended-JDK8-win_x86-64_cmprssptrs`
-    - Trigger:
-        - build periodically, @midnight
-
-- Pipeline-Build-Test-JDK9-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Pipeline-Build-Test-JDK9-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Pipeline-Build-Test-JDK9-win_x86-64_cmprssptrs)
-    - Description:
-        - Compile and Test java9 Sanity & Extended
-        - Triggers
-            - `Build-JDK9-win_x86-64_cmprssptrs`
-            - `Test-Sanity-JDK9-win_x86-64_cmprssptrs`
-            - `Test-Extended-JDK9-win_x86-64_cmprssptrs`
-    - Trigger:
-        - build periodically, @midnight
-
-- Pipeline-Build-Test-JDK10-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Pipeline-Build-Test-JDK10-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Pipeline-Build-Test-JDK10-win_x86-64_cmprssptrs)
-    - Description:
-        - Compile and Test java10 Sanity & Extended
-        - Triggers
-            - `Build-JDK10-win_x86-64_cmprssptrs`
-            - `Test-Sanity-JDK10-win_x86-64_cmprssptrs`
-            - `Test-Extended-JDK10-win_x86-64_cmprssptrs`
+        - Compile Eclipse OpenJ9 on `<platform>` for Extensions OpenJDK`<version>` and run sanity & extended test suites 
+        - Triggers:
+            - Build-JDK`<version>`-`<platform>`
+            - any platform and version except win_x86 and win_x86-64_cmprssptrs
+               - Test-`<sanity|extended>.<functional|system>`-JDK`<version>`-`<platform>`
+            - win_x86 and win_x86-64_cmprssptrs any version
+                - Test-`<Sanity|Extended>`-JDK`<version>`-`<platform>`
```

Comment:
I think since the win exception is temporary, we should just ignore it and only talk about X.Y.